cpp/throwaways/generate_gnd_association_and_spelling_replacements.sh

```diff
@@ -0,0 +1,93 @@
+# Create krimdok local keyword file from the original output and make misspellings and potential GND references explicit
+#!/bin/bash
+
+
+if [ $# != 4 ]; then
+    echo "usage: $0 A-Z_output A_Z_bereinigt gnd_full.mrc result"
+    exit 1
+fi
+
+readonly AZ_OUTPUT=$1
+readonly AZ_CLEANED=$2
+readonly GND_FULL=$3
+readonly RESULT_FILE=$4
+readonly tmp_prefix=$(basename $0)
+#readonly gnd_matches="./matched3.txt"
+#readonly gnd_matches="./tmp/matched_210609_1.txt"
+
+function MkTempFile {
+  echo $(mktemp /tmp/${tmp_prefix}.XXXXX)
+}
+
+
+function GenerateKeywordToGNDMap {
+   readonly gnd_matched=$(MkTempFile)
```

Comment:
These TempFiles should also be defined outside this function, because they are re-used in other functions later